web/src/services/api.ts

```diff
@@ -105,7 +105,7 @@ export default class API {
     return this.fetch(`${this.getLocalePath()}/sentences?count=${count}`);
   }
 
-  fetchRandomClips(count: number = 1): Promise<Clip[]> {
+  async fetchRandomClips(count = 1): Promise<Clip[]> {
```

Comment:
Adding `async` here is actually important because I'm not sure the errors were being caught otherwise